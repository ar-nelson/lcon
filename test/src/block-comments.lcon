#:
  This is a top-level block comment.
test1
  foo bar #: baz
test2
  foo bar #: baz
    a b
      c d
  e f
test3
  foo bar
  a b #:
    c d
  e f
