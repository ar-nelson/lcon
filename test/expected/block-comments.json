{
  "test1": {"foo": "bar"},
  "test2": {
    "foo": "bar",
    "e": "f"
  },
  "test3": {
    "foo": "bar",
    "a": "b",
    "e": "f"
  }
}
